src/main/java/org/programmers/voucher/io/ConsoleInput.java

```diff
@@ -0,0 +1,38 @@
+package org.programmers.voucher.io;
+
+import org.programmers.voucher.domain.VoucherType;
+import org.programmers.voucher.util.Command;
+import org.programmers.voucher.util.IllegalCommandException;
+import org.programmers.voucher.util.IllegalVoucherTypeException;
+
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.Scanner;
+
+public class ConsoleInput implements Input{
+    private final Scanner sc = new Scanner(System.in);
+
+    @Override
+    public Command inputCommand() throws IllegalCommandException {
+        String input = sc.nextLine().strip();
+        Optional<Command> inputCommand = Arrays.stream(Command.values())
+                .filter(value -> value.toString().equals(input))
```

Comment:
```suggestion
	    .filter(value -> value.isMatches(input))
```

이런 방식으로 command라는 필드를 toString으로 가져와서 비교하는게 아니라 enum에 비교문을 만들어놓고 api로 제공하는 것이 메시징의 의미를 부여할 수 있을 것 같습니다.